tests/Microsoft.Identity.Web.Test/TokenAcquisitionAddInTests.cs

```diff
@@ -27,33 +32,61 @@ public void InvokeOnBuildConfidentialClientApplication_InvokesEvent()
             // Assert
             Assert.True(eventInvoked);
         }
-
+#endif
         [Fact]
-        public void InvokeOnBeforeTokenAcquisitionForApp_InvokesEvent()
+        public async Task InvokeOnBeforeTokenAcquisitionForApp_InvokesEvent()
```

Comment:
Had a hard time mocking MSAL components because of MSAL's implementation. Using the Http mocking infrastructure instead.